backend/docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+db:
+  image: postgres:9.5
+web:
+  build: .
+  command: python manage.py runserver 0.0.0.0:8000
+  volumes:
+    - .:/opt/akvo-dash
```

Comment:
maybe I'm missing something... but why do we need the current host path mounted at /opt/akvo-dash?